cvat/apps/documentation/user_guide.md

```diff
@@ -252,81 +467,81 @@ Usage examples:
 
 ---
 **Annotation** is a set of shapes and tracks. There are several types of annotations:
-- *Manual* which is created by a person
-- *Semi-automatic* which is created mainly automatically but the user provides some data (e.g. interpolation)
-- *Automatic* which is created automatically without a person in the loop
-
-## Interface of the annotation tool
-
-The tool is composed of:
-- ``Workspace`` — where images are shown;
-- ``Bottom panel`` (under workspace) — for navigation, filtering annotation and accessing tools' menu;
-- ``Side panel`` — contains two lists: of objects (on the frame) and labels (of objects on the frame);
-- ``Bottom side panel`` — for choosing types of/creating/merging/grouping annotation;
-
-![](static/documentation/images/image034.jpg)
-
-There is also:
-- ``Settings`` (F2) — pop-up in the Bottom panel, contains different parameters which can be adjusted by the user's needs
-
-- ``Context menu`` — available on right mouse button.
+- _Manual_ which is created by a person
+- _Semi-automatic_ which is created mainly automatically, but the user provides some data (e.g. interpolation)
+- _Automatic_ which is created automatically without a person in the loop
 
 ---
 ### Workspace — Context menu
 
 Context menu opens by right mouse click.
```

Comment:
Is it correct sentence? Could you please check. I would expect passive voice here.